docs/en/event-daemon/event-daemon.md

```diff
@@ -4,3 +4,65 @@ title: Writing Event-Driven Triggers
 pagename: event-daemon
 lang: en
 ---
+
+[![Python 2.6 2.7 3.7](https://img.shields.io/badge/python-2.6%20%7C%202.7%20%7C%203.7-blue.svg)](https://www.python.org/)
+[![Build Status](https://dev.azure.com/shotgun-ecosystem/Toolkit/_apis/build/status/shotgunEvents?branchName=master)](https://dev.azure.com/shotgun-ecosystem/Toolkit/_build/latest?definitionId=89&branchName=master)
+[![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+[![Linting](https://img.shields.io/badge/PEP8%20by-Hound%20CI-a873d1.svg)](https://houndci.com)
```

Comment:
Remove all of these. They're relevant for the github repo, so we'll end up keeping them there, but it's not needed in these docs.